### DISCUSSION TITLE
History atom with structured details

### DISCUSSION THREAD

**Discussion by gms8994** [trakt/trakt-api#325] [Q&A] [answered]
I know about the user history.atom endpoint (which is awesome). What I'd love to have is some kind of parameter that could be passed that would provide a structured result for things like Episodes (that would match the output of all of the "show" endpoints) or with Movies. Maybe something like that already exists?

**Comment by rectifyer** [comment, marked as answer]
We don't plan on offering structured XML, I suggest using the JSON based Trakt API instead. Is that possible in your project?
